### PR TITLE
Merge/1.7 into master

### PR DESCRIPTION
**Pull request by hilmarf**
see #1021

### CODE REVIEW

File: services/distribution/src/main/resources/master-config/v2/risk-calculation-parameters.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ minutes-at-attenuation-filters:
       min: 0
       max: 73
       max-exclusive: true
+  -
```
Comment (hilmarf) [inline services/distribution/src/main/resources/master-config/v2/risk-calculation-parameters.yaml:16]: @mlenkeit this additional `-` is correct, right?

**Review by sorin-iovita — DISMISSED** [review]

File: services/distribution/src/main/resources/master-config/v2/risk-calculation-parameters.yaml (hunk 1 above)
Comment (mlenkeit) [reply services/distribution/src/main/resources/master-config/v2/risk-calculation-parameters.yaml:16]: no, thanks for catching this!

**Review by mlenkeit — CHANGES_REQUESTED** [review]
see https://github.com/corona-warn-app/cwa-server/pull/1023/files#r532390312

**Review by mlenkeit — APPROVED** [review]
LGTM

**Review by fredrb — APPROVED** [review]

**Review by sorin-iovita — APPROVED** [review]

**Pull request merged**